platform-dom/src/main/java/com/softicar/platform/dom/elements/popup/configuration/DomPopupDisplayMode.java

```diff
@@ -13,42 +14,71 @@ public enum DomPopupDisplayMode {
 	 * A modal, dismissable, non-draggable {@link DomPopup} without header.
 	 * Provides a visual clue to its modality.
 	 */
-	DIALOG(false, DomPopupModalMode.MODAL_DISMISSABLE),
+	DIALOG(false, false, true, DomPopupModalMode.MODAL_DISMISSABLE, DomPopupChildClosingMode.INTERACTIVE_ALL),
```

Comment:
wow, three boolean parameters in a row?

Reply:
It kind of just happened. I think this could be refactored to not be enum-based... there should be setters for those flags, etc. But I considered this low-prio, and a candidate for subsequent cleanup, because the CTOR is purely internal (because obviously private) anyway.

Reply:
see my other comment about the complexity, we should cut this down or we will get problems later to upkeep this whole complexity

Reply:
You mean, like... now? Again, this is purely internal, and the issues in legibility due to the ugly CTOR are restricted to just 5 LOC in this enum.
If you insist, on changing this now, pls provide a concrete proposal on how to refactor this enum.